src/Features/LanguageServer/Protocol/Handler/Initialize/InitializeHandler.cs

```diff
@@ -27,6 +27,7 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
                 DocumentRangeFormattingProvider = true,
                 DocumentOnTypeFormattingProvider = new DocumentOnTypeFormattingOptions { FirstTriggerCharacter = "}", MoreTriggerCharacter = new[] { ";", "\n" } },
                 DocumentHighlightProvider = true,
+                TextDocumentSync = new TextDocumentSyncOptions { OpenClose = true }
```

Comment:
Do we need this since the notifications aren't propagated through live share yet?